src/nvim/window.c

```diff
@@ -5222,10 +5222,16 @@ void win_setheight_win(int height, win_T *win)
     if (full_screen && msg_scrolled == 0 && row < cmdline_row) {
       grid_fill(&default_grid, row, cmdline_row, 0, Columns, ' ', ' ', 0);
     }
+    if (msg_grid.chars) {
```

Comment:
wouldn't this be nested inside the previous `if` (the space was increased) ?

Reply:
ah, good catch, fixed it